kratos/includes/linear_master_slave_constraint.h

```diff
@@ -387,6 +387,43 @@ class LinearMasterSlaveConstraint
         return mMasterDofsVector;
     }
 
+    /**
+     * @brief This method resets the values of the slave dofs
+     * @param rCurrentProcessInfo the current process info instance
+     */
+    void ResetSlaveDofs(const ProcessInfo& rCurrentProcessInfo) override
+    {
+        for (IndexType i = 0; i < mSlaveDofsVector.size(); ++i) {
+            #pragma omp atomic
+            mSlaveDofsVector[i]->GetSolutionStepValue() *= 0.0;
```

Comment:
just curious, is this faster than `mSlaveDofsVector[i]->GetSolutionStepValue() = 0.0;` ?

Reply:
I assume so, because critical is killer

Reply:
@philbucher it is not faster, but there is no atomic assignement (at least in openmp 2.5)

Reply:
ok thx good to know